Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -6,10 +6,11 @@
   align-items: center;
 }
 
-.bocCheckBox input
+.bocCheckBox input[type=checkbox]
 {
   position: absolute;
   z-index: var(--z-index-hidden);
+  opacity: 0;
```

Comment:
You might want to use `visibility: hidden` here, even if it does the same thing.

Reply:
No unfortunately they are not doing the same thing. With `visibility: hidden` the checkbox/radiobutton is no longer tabbable.

Reply:
Interesting, then ignore the 2 comments about it, sorry about that.